src/main/java/com/server/bugtracker/User/User.java

```diff
@@ -71,12 +71,12 @@ public void setName(String name)
 
     public String getUser_name()
     {
-        return user_name;
```

Comment:
Your name change ("username") will not work with the current database set up. Please remove your changes or explain why the change was made.